### PR TITLE
Added Vagrant description

### PR DESCRIPTION
**Pull request by alexkorsun**
- added description for Vagrant
- updated gitignore to not commit idea files when use Webstorm IDE

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@raisedadead Check out the change to .gitignore

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 node_modules
 
+/.idea
```
Comment (raisedadead) [inline .gitignore]:
```suggestion
.idea
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
A minor change.

**Comment by raisedadead** [comment]
@RandellDawson We should consolidate all the .gitignore files together from the different sub projects.

**Comment by raisedadead** [comment]
Closing as stale/decayed.

Hi,

Thanks a lot for your pull request. We have not seen an update from you on this PR in a while. We are going to have to close this as stale. If you are still interested, please open a fresh PR.

In case you believe this is closed incorrectly please hit us up in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors) to get this re-opened.

Thanks and happy contributing.

**Pull request closed without merging**